areas/appservice/src/AzureMcp.AppService/Commands/Database/DatabaseAddCommand.cs

```diff
@@ -0,0 +1,92 @@
+        settings for the specified App Service, allowing it to connect to a database server.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new() { Destructive = false, ReadOnly = false };
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        command.AddOption(AppServiceOptionDefinitions.AppServiceName);
+        command.AddOption(AppServiceOptionDefinitions.DatabaseTypeOption);
+        command.AddOption(AppServiceOptionDefinitions.DatabaseServerOption);
+        command.AddOption(AppServiceOptionDefinitions.DatabaseNameOption);
+        command.AddOption(AppServiceOptionDefinitions.ConnectionStringOption);
+    }
+
+    protected override DatabaseAddOptions BindOptions(ParseResult parseResult)
+    {
+        var options = base.BindOptions(parseResult);
+        options.AppName = parseResult.GetValueForOption(AppServiceOptionDefinitions.AppServiceName);
+        options.DatabaseType = parseResult.GetValueForOption(AppServiceOptionDefinitions.DatabaseTypeOption);
+        options.DatabaseServer = parseResult.GetValueForOption(AppServiceOptionDefinitions.DatabaseServerOption);
+        options.DatabaseName = parseResult.GetValueForOption(AppServiceOptionDefinitions.DatabaseNameOption);
+        options.ConnectionString = parseResult.GetValueForOption(AppServiceOptionDefinitions.ConnectionStringOption);
+        return options;
+    }
+
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var options = BindOptions(parseResult);
+
+        try
+        {
+            if (!Validate(parseResult.CommandResult, context.Response).IsValid)
+            {
+                return context.Response;
+            }
+
+            context.Activity?.WithSubscriptionTag(options);
+
+            var appServiceService = context.GetService<IAppServiceService>();
+            var connectionInfo = await appServiceService.AddDatabaseAsync(
+                options.AppName!,
+                options.ResourceGroup!,
+                options.DatabaseType!,
+                options.DatabaseServer!,
+                options.DatabaseName!,
+                options.ConnectionString ?? string.Empty, // connectionString - will be generated if not provided
+                options.Subscription!,
+                options.Tenant,
+                options.RetryPolicy);
+
+            context.Response.Results = ResponseResult.Create(
+                new Result(connectionInfo),
+                AppServiceJsonContext.DatabaseAddCommandResult);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Failed to add database connection to App Service '{AppName}'", options.AppName);
+            HandleException(context, ex);
+        }
+
+        return context.Response;
+    }
+
+    public record Result(DatabaseConnectionInfo ConnectionInfo);
```

Comment:
We've been making the names of these records pretty verbose/descriptive. Should we use Result instead @jongio? After all, if we ever use them in other classes we could do so by using `DatabaseAddCommand.Result`, for example.